package.json

```diff
@@ -30,6 +30,7 @@
     "remark-stringify": "^7.0.4",
     "unified": "^8.4.2",
     "unist-util-inspect": "^5.0.0",
+    "url": "^0.11.0",
```

Comment:
Could we do this with the built-in URL object? https://nodejs.org/api/url.html

Reply:
That was what I wanted myself, but it just ended up getting messy, especially since I don't really program in JavaScript. I kept wishing I had the Python libs at my disposal! :) Core to the idea is to reuse the relative to absolute path conversion from URL, which is the most uprated solution in StackOverflow (https://stackoverflow.com/questions/14780350/convert-relative-path-to-absolute-using-javascript). There are always some corner cases that makes custom functions more unreliable. 

Also, I do not have time to do that myself anyway. URL should be a safe dependency, I believe? I feel the code at least became quite clean in this way.

Reply:
But now I am confused. Isn't it the built-in url module I used? It supports both the deprecated Legacy API and the WHATWG API. Or should you not need to import that?

Reply:
I thought you meant a custom-written function above. Please inform me!

Reply:
You can use

```javascript
const url = require('url');
```

in your JS code without adding anything in `package.json` and the built-in module will be used.